dev_guide/downward_api.adoc

```diff
@@ -222,3 +219,139 @@ annotation2=456
 kubernetes.io/config.source=api
 ----
 ====
+
+[[dapi-consuming-container-resources]]
+== Consuming Container Resources Using the Downward API
+
+When creating pods, you can use the downward API to inject information about
+computing resource requests and limits so that image and application authors can
+correctly create an image for specific environments.
+
+You can do this using both the xref:dapi-using-environment-variables[environment variable] and xref:dapi-using-volume-plugin[volume plug-in] methods.
+
+[[dapi-resources-environment-variables]]
+=== Using Environment Variables
+
+. When creating a pod configuration, specify environment variables that
+correspond to the contents of the `resources` field in the `spec.container`
+field:
++
+====
+----
+...
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox:1.24
+      command: [ "/bin/sh", "-c", "env" ]
+      resources:
+        requests:
+          memory: "32Mi"
+          cpu: "125m"
+        limits:
+          memory: "64Mi"
+          cpu: "250m"
+      env:
+        - name: MY_CPU_REQUEST
+          valueFrom:
+            resourceFieldRef:
+              resource: requests.cpu
+        - name: MY_CPU_LIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.cpu
+        - name: MY_MEM_REQUEST
+          valueFrom:
+            resourceFieldRef:
+              resource: requests.memory
+        - name: MY_MEM_LIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.memory
+...
+----
+====
++
+If the resource limits are not included in the container configuration, the downward API defaults to the node's CPU and memory capacities.
```

Comment:
node's CPU and memory allocatables

It was changed in kubernetes/kubernetes#29639 and I also sent a doc update PR here:
https://github.com/kubernetes/kubernetes.github.io/pull/1107/files